### PR TITLE
fix UTF-8 and characters mess

### PR DESCRIPTION
**Pull request by KurtMi**
part of PR split

### CODE REVIEW

File: doc/NEWS.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ Thanks to Manuel Mausz for to testing and improving portability!
 - make dbus example work again
 - fix compiler warnings for gcc and clang
 - fix valgrind suppressions
-- Installation of GI binding is fixed, thanks to Dāvis
+- Installation of GI binding is fixed, thanks to Davis
```
Comment (markus2330) [inline doc/NEWS.md]: Other parts of this PR is ok, but I think thats a name written like that. This should not be changed to ASCII.

Why exactly is this change needed?

**Comment by KurtMi** [comment]
How do we resolve this conflicts?

**Comment by KurtMi** [comment]
I have rebased the branch with master and resolved the conflicts, by taking every time the HEAD content and removing the blanks at end of line. I checked also the pdf output.

**Comment by markus2330** [comment]
Sorry, forgot about this PR! Thanks for fixing the whitespaces!
